src/vitafile.cpp

```diff
@@ -213,7 +213,7 @@ std::vector<uint8_t> pkgi_load(const std::string& path)
 void pkgi_save(const std::string& path, const void* data, uint32_t size)
 {
     SceUID fd = sceIoOpen(
-            path.c_str(), SCE_O_WRONLY | SCE_O_CREAT | SCE_O_TRUNC, 0777);
+            path.c_str(), SCE_O_WRONLY | SCE_O_CREAT /* | SCE_O_TRUNC */, 0777);
```

Comment:
Not sure about this change, if we pkgi_save on a file that already exists, it will only overwrite the beginning of it. This might break things

Reply:
you can revert this it was a bug ii was having where it wasnt writing the rif it ended up being because it was passibg an empty vector<int8> to the array. Im not actually sure what SCE_O_TRUNC does tbh i just tried editing it as thats what my bgdl example code used and it didnt fix abything as that open wasnt actuslly yhe problem. And i just never reverted it when i did end up fixing it